src/main/java/com/currencycloud/client/model/Iban.java

```diff
@@ -134,20 +140,36 @@ public void setUpdatedAt(Date updatedAt) {
         this.updatedAt = updatedAt;
     }
 
+    public String getScope() {
+        return scope;
+    }
+
+    public void setScope(String scope) {
+        this.scope = scope;
+    }
+
     @Override
     public String toString() {
-        return new JSONObject()
-                .appendField("id", id)
-                .appendField("ibanCode", ibanCode)
-                .appendField("accountId", accountId)
-                .appendField("currency", currency)
-                .appendField("accountHolderName", accountHolderName)
-                .appendField("bankInstitutionName", bankInstitutionName)
-                .appendField("bankInstitutionAddress", bankInstitutionAddress)
-                .appendField("bankInstitutionCountry", bankInstitutionCountry)
-                .appendField("bicSwift", bicSwift)
-                .appendField("createdAt", createdAt)
-                .appendField("updatedAt", updatedAt)
-                .toString();
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
objectMapper should really be declared "final", also map below

Reply:
Agreed. Will implement as suggested